### PR TITLE
Fixing compile error on Unreal 5.1 Preview

### PR DESCRIPTION
**Pull request by Edstub207**

### CODE REVIEW

**Comment by Edstub207** [comment]
@tustanivsky FYI

File: plugin-dev/Source/SentryEditor/Private/SentrySettingsCustomization.cpp [hunk 1]
```diff
@@ -43,8 +43,8 @@ void FSentrySettingsCustomization::DrawDebugSymbolsNotice(IDetailLayoutBuilder&
 				[
 					SNew(SRichTextBlock)
 						.Text(FText::FromString(TEXT("Note that the Sentry SDK creates a <RichTextBlock.TextHighlight>sentry.properties</> file at project root to store the configuration, that should <RichTextBlock.TextHighlight>NOT</> be made publicly available.")))
-						.TextStyle(FEditorStyle::Get(), "MessageLog")
-						.DecoratorStyleSet(&FEditorStyle::Get())
```
Comment (bitsandfoxes) [inline plugin-dev/Source/SentryEditor/Private/SentrySettingsCustomization.cpp:47]: Would that be backward compatible?

Comment (Edstub207) [reply plugin-dev/Source/SentryEditor/Private/SentrySettingsCustomization.cpp:47]: @bitsandfoxes in UE5, yes. Doesn't look like it on UE4.

**Comment by tustanivsky** [comment]
The issues mentioned above were addressed in #150, closing it now

**Pull request closed without merging**